### ISSUE TITLE
J / K keyboard navigation does not work in Safari 6

### ISSUE THREAD

**Issue by ticky** [jcutler/Missing-E#35]
Pressing J or K immediately switches to the next / previous page, respectively, no matter where the viewport is on the page.

Is there any good way for me to provide better information for your diagnosis?
